.github/workflows/docs.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy docs
+
+on:
+  push:
+    branches:
+      master
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Bootstrap
+      uses: wemeetagain/actions-lerna@v0.1.0
+      with:
+        cmd: bootstrap
+    - name: Spec tests
+      uses: wemeetagain/actions-lerna@v0.1.0
+      with:
+          cmd: run run build:release
+    - name: Build docs
+      run: node scripts/collect-docs.js
+    - name: Deploy
+      uses: peaceiris/actions-gh-pages@v2
+      env:
+        PERSONAL_TOKEN: ${{ secrets.GH_PAGES_TOKEN }}
```

Comment:
How's this getting triggered? I don't see any variable named GH_PAGES_TOKEN in the settings.

Reply:
you're right, I guess that was set on travis's secrets.

Reply:
yeah so, for github actions, you have to set the secrets in the individual repo

Reply:
can't you put secrets in repo -> settings -> secrets?

Isn't repo just for large secrets?

Reply:
added token